recipes/lemon/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "lemon" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "71b7c725f4c0b4a8ccb92eb87b208701586cf7a96156ebd821ca3ed855bad3c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://lemon.cs.elte.hu/pub/sources/lemon-{{ version  }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - glpk_win.diff    # [win]
```

Comment:
Anyways once that issue is raised and a comment added to this recipe with a link to that issue, I'd be more than happy to merge.